.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,7 +23,6 @@
 - [ ] (If applicable) Did you regenerate and publicly share default AGFIs?
 <!-- Do this if this PR is a bugfix that should be applied to master -->
 - [ ] (If applicable) Did you mark the PR as "Please Backport"?
-- [ ] (On merge) Did you update release notes in the dev-to-master PR ?
```

Comment:
I assume that these need to have alternate suggestions and we also need to remove the `master` in the comment above this too.

Reply:
Yea, I think the comment needs to change to something like `Do this if this PR is a bugfix that should be applied to the latest release`. The other bullets seem reasonable still.